code/game/turfs/simulated.dm

```diff
@@ -171,3 +171,20 @@
 	if(ticker && ticker.current_state == GAME_STATE_PLAYING)
 		fluid_update()
 	. = ..()
+
+/turf/simulated/verb/engrave()
+	set name = "Engrave floor"
+	set desc = "Writes a message on the floor."
+	set category = "Object"
+	set src in oview(1)
```

Comment:
Haven't gotten around to testing this yet. Shouldn't it just be view(1)?

Reply:
It should work as-is if I remember correctly. I just used oview since that’s what table flipping used, and was the thing I looked at for how to do verbs. I can change it and see if it still works though, if you’d prefer.

Reply:
Its use in table flipping is because you can't flip something you're standing on.

Reply:
I’m not sure it’s the set that does that, there seems to be other stuff that handles that. Plus, quite a few other verbs use it too, even stuff that seems like it wouldn’t matter if it’s on the same tile as you.

Reply:
As much as it doesn't make any sense to me, this somehow still works.